spec/facility_spec.rb

```diff
@@ -2,25 +2,193 @@
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('Albany DMV Office')
-      expect(@facility.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
-      expect(@facility.phone).to eq('541-967-2014')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('Albany DMV Office')
+      expect(@facility_1.address).to eq('2242 Santiam Hwy SE Albany OR 97321')
+      expect(@facility_1.phone).to eq('541-967-2014')
+      expect(@facility_1.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@cruz.registration_date).to eq(nil)
+      expect(@facility_1.registered_vehicles).to eq([])
+      expect(@facility_1.collected_fees).to eq(0)
+    end
+  end
+
+  it 'registers cruz car' do
+    expect(@facility_1.add_service('Vehicle Registration')).to eq(['Vehicle Registration'])
+    expect(@cruz.registration_date).to eq(nil)
+    expect(@facility_1.registered_vehicles).to eq([])
+    expect(@facility_1.collected_fees).to eq(0)
+
+    expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+    expect(@cruz.registration_date).to eq(Date.today)
+    expect(@cruz.plate_type).to eq(:regular)
+    expect(@facility_1.registered_vehicles).to eq([@cruz])
+    expect(@facility_1.collected_fees).to eq(100)
+  end
+
+  it 'registers camaro car' do
+    expect(@facility_1.add_service('Vehicle Registration')).to eq(['Vehicle Registration'])
+    expect(@cruz.registration_date).to eq(nil)
+    expect(@facility_1.registered_vehicles).to eq([])
+    expect(@facility_1.collected_fees).to eq(0)
+
+    expect(@facility_1.register_vehicle(@cruz)).to eq([@cruz])
+    expect(@cruz.registration_date).to eq(Date.today)
+    expect(@cruz.plate_type).to eq(:regular)
+    expect(@facility_1.registered_vehicles).to eq([@cruz])
+    expect(@facility_1.collected_fees).to eq(100)
+
+    expect(@facility_1.register_vehicle(@camaro)).to eq([@cruz, @camaro])
+    expect(@camaro.registration_date).to eq(Date.today)
+    expect(@camaro.plate_type).to eq(:antique)
+    expect(@facility_1.registered_vehicles).to eq([@cruz, @camaro])
+    expect(@facility_1.collected_fees).to eq(125)
+  end
+
+  it 'registers bolt car' do
```

Comment:
Your tests themselves are good, I would just like to see these it blocks encompassing particular behaviors of methods, not being focused on specific instances. So this it block could be "it registers :regular vehicle", but you could also combine these three tests. (for the three cars) since they are so similar and have the it block be `it "registers vehicles"`